_sass/components/_library-card.scss

```diff
@@ -0,0 +1,78 @@
+// Library card ------------- //
+
+// Sets a height on library entries so they're the same height and don't break the grid
+.library-card {
+  box-shadow: 0 0 0 1px $color-gray-lighter;
+  display: block;
+  margin-bottom: 3rem;
+  padding-bottom: 3rem;
+  position: relative;
+  text-decoration: none;
+
+  @include media($medium-screen) {
+    display: flex;
+    flex-direction: column;
+    margin-right: 2.3%;
+    min-height: 350px;
```

Comment:
@wslack at the same time, can we add in the color-box header that we discussed briefly yesterday? so that they appear to have more definition (honestly, they will likely look a lot like monopoly cards - but all the way across the top, not individual boxes on each). Thoughts?

![image](https://user-images.githubusercontent.com/25348927/28585942-046d7092-7140-11e7-85b4-036fed6802ee.png)